python/ceed_vector.py

```diff
@@ -17,6 +17,7 @@
 from _ceed_cffi import ffi, lib
 import tempfile
 import numpy as np
+import numba.cuda as nbcuda
```

Comment:
Can we make this optional, such that one need not have Numba (or CUDA) to use libCEED, but if they use Numba, it will work?

Reply:
sure, I'll look into that

Reply:
How about the adjustment I made?